### PR TITLE
Refactoring network manager

### PR DESCRIPTION
**Pull request by igorkulman**
Some cleanup in the `NetworkManager`class to make the methods readable, file structure, using image literals.

### CODE REVIEW

**Comment by igorkulman** [comment]
Fixed conflicts with the latest master branch.

File: HeliPort/NetworkManager+Data.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+final class NetworkInfo {
+    let ssid: String
+    let isConnected: Bool
+    var rssi: Int
+
+    var auth = NetworkAuth()
+
+    enum AuthSecurity: UInt32 {
+        case NONE        = 0x00000000
+        case USEGROUP    = 0x00000001
```
Comment (Goshin) [inline HeliPort/NetworkManager+Data.swift:27]: We don't need the `AuthSecurity` enum anymore. Btw, can you put the description extension of `itl80211_security` enum together with that of `itl_phy_mode` enum?

File: HeliPort.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -65,16 +68,16 @@
 
 /* Begin PBXFileReference section */
 		2B879D3A8D737E5EF06C350F /* Pods-HeliPort.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-HeliPort.release.xcconfig"; path = "Target Support Files/Pods-HeliPort/Pods-HeliPort.release.xcconfig"; sourceTree = "<group>"; };
+		42979EFF24B3A27200F15FBD /* el */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = el; path = el.lproj/Localizable.strings; sourceTree = "<group>"; };
```
Comment (Goshin) [inline HeliPort.xcodeproj/project.pbxproj:71]: Before I merge it in, can you remove the unnecessary change here? It might cause conflict with other PRs.

File: HeliPort.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -65,16 +68,16 @@
...
 		50B86ABC24B22F75008E4FE4 /* zh-Hans */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = "zh-Hans"; path = "zh-Hans.lproj/Localizable.strings"; sourceTree = "<group>"; };
 		50B86ABD24B22FEE008E4FE4 /* pt-BR */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = "pt-BR"; path = "pt-BR.lproj/Localizable.strings"; sourceTree = "<group>"; };
 		6A54F80824B24B9700B8D651 /* it */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = it; path = it.lproj/Localizable.strings; sourceTree = "<group>"; };
 		6A78687824B26AF6007832EE /* tr */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = tr; path = tr.lproj/Localizable.strings; sourceTree = "<group>"; };
-		42979EFF24B3A27200F15FBD /* el */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = el; path = el.lproj/Localizable.strings; sourceTree = "<group>"; };
```
Comment (Goshin) [inline HeliPort.xcodeproj/project.pbxproj:74]: and here

File: HeliPort.xcodeproj/project.pbxproj (hunk 2 above)
Comment (igorkulman) [reply HeliPort.xcodeproj/project.pbxproj:71]: The reference id is the same so no conflict. Moving lines around in `pbxproj` is just the crap Xcode does sometimes. I have a suspicion that this file was edited manually and Xcode just tries to "correct" it.

Comment (Goshin) [reply HeliPort.xcodeproj/project.pbxproj:71]: OK. Thanks for the great work.
